spaces/domains.json

```
@@ -757,5 +757,6 @@
   "vote.nation3.org": "nation3.eth",
   "dao.hausofprime.com": "hausofprime.eth",
   "vote.famdao.org": "fam-dao.eth",
-  "gov.daomaker.com": "shomustgoon.eth"
+  "gov.daomaker.com": "shomustgoon.eth",
+  "vote.fstswap.exchange","fistdex.eth"
```

You will need to replace the comma with a colon.
```suggestion
  "vote.fstswap.exchange": "fistdex.eth"
```